README.md

```diff
@@ -83,10 +78,7 @@ mcollective class, with secondary configuration managed by the defined types
 
 ```puppet
 node 'broker1.example.com' {
-  class { '::mcollective':
-    middleware       => true,
-    middleware_hosts => [ 'broker1.example.com' ],
-  }
+  include activemq
```

Comment:
In this example, it's just a broker, so just a place where activemq goes.  That's what the previous code did, it wouldn't install mcollective server or client components, just manage the middleware onto this host.